server/server.go

```diff
@@ -290,6 +298,19 @@ func ReadCommitmentMode(r *http.Request) (commitments.CommitmentMode, error) {
 	return commitments.OptimismAltDA, nil
 }
 
+func ReadCommitmentVersion(r *http.Request, mode commitments.CommitmentMode) (uint8, error) {
+	commitment := r.URL.Path
+	if len(commitment) < 3 {
+		return 0, fmt.Errorf("commitment is too short")
+	}
+
+	if mode == commitments.OptimismAltDA || mode == commitments.OptimismGeneric {
+		return commitment[2], nil
+	}
+	// the only other mode is simple, which take first byte as version
+	return commitment[0], nil
+}
```

Comment:
By commitment version do we mean certificate version?

Reply:
yep, I can update to certificate version for clarify